storm/__init__.py

```diff
@@ -27,18 +27,23 @@ def __init__(self, ssh_config_file=None):
         self.ssh_config.load()
         self.defaults = self.ssh_config.defaults
 
-    def add_entry(self, name, host, user, port, id_file, custom_options=[]):
+    def add_entry(self, name, host, user, port, id_file,
+                  custom_options=[], dry_run=None):
         if self.is_host_in(name):
             raise ValueError(ERRORS["already_in"].format(name))
 
         options = self.get_options(host, user, port, id_file, custom_options)
 
         self.ssh_config.add_host(name, options)
+
+        if dry_run:
+            return self.ssh_config.dump()
+
```

Comment:
Mmm yes, I missed the return before `self.ssh_config.dump()`. I thought that made the dry run kinda pointless. Makes sense now :)